src/api/status/public/js/feed/index.js

```diff
@@ -0,0 +1,12 @@
+const { fetchFeeds } = require('../api');
+
+export async function getFeedCount() {
+  try {
+    const data = await fetchFeeds();
```

Comment:
Because all you care about are the headers, use `fetch(url, {method: 'HEAD'})` to reduce how much gets sent back.